starkbank/merchantcard/__merchantcard.py

```diff
@@ -0,0 +1,52 @@
+from starkcore.utils.resource import Resource
+from ..utils import rest
+from starkcore.utils.checks import check_date, check_datetime, check_datetime_or_date
+
+class MerchantCard(Resource):
+    """# MerchantCard object
+    Check out our API Documentation at https://starkbank.com/docs/api#merchant-card
+    """
+
+    def __init__(self, id=None, ending=None , funding_type=None, holder_name=None, network=None, status=None, tags=None, expiration=None, created=None, updated=None):
+        Resource.__init__(self, id=id)
+        self.ending = ending
+        self.funding_type = funding_type
+        self.holder_name = holder_name
+        self.network = network
+        self.status = status
+        self.tags = tags
+        self.expiration = check_datetime_or_date(expiration)
+        self.created = check_datetime(created)
+        self.updated = check_datetime(updated)
+
+_resource = {"class": MerchantCard, "name": "MerchantCard"}
+
```

Comment:
Faz isso nos outros lugares que você codou também
```suggestion


_resource = {"class": MerchantCard, "name": "MerchantCard"}


```